app/assets/javascripts/project.js

```diff
@@ -26,6 +26,12 @@ document.addEventListener("turbolinks:load", function () {
       stories_ids.push($(checkbox).val());
     });
 
+    const ending = stories_ids.length == 1 ? "y" : "ies";
+    let user_confirmation = confirm(
+      `Are you sure you want to bulk delete ${stories_ids.length} stor${ending}?`
```

Comment:
What do you think of this?
```suggestion
      `Are you sure you want to delete ${stories_ids.length} stor${ending}?`
```

Reply:
That sounds way better, should've worded it like that !! 😄😄